data-explorer/kusto/concepts/data-soft-delete.md

```diff
@@ -42,6 +42,7 @@ The soft delete process is performed using the following steps:
 * Soft delete is executed against your engine endpoint: `https://[YourClusterName].[region].kusto.windows.net`. The command requires [database admin](../management/access-control/role-based-access-control.md) permissions on the relevant database.
 
 * Soft delete can affect materialized views based on a source table in which records are deleted. This can happen because every [materialization cycle](../management/materialized-views/materialized-view-overview.md#how-materialized-views-work) adds newly ingested data to the materialized part from the previous cycle. Therefore, if the command deletes newly ingested records before a new cycle begins, those records won't be added to the materialized view. Otherwise, deleting records won't affect the materialized view.
```

Comment:
I would add a `##` section for `Soft delete for table`, just like you added for materialized views. Add this bullet about how soft delete affects MVs under that section (it's only relevant when deleting from the source table of an MV, not when deleting from the MV itself). 
Additionally, you can remove the note below ("soft delete is also supported for materialized views")